source/fundamentals/typescript.txt

```diff
@@ -142,6 +142,60 @@ section.
 
 .. _node-driver-limitations:
 
+Find Methods and the _id Field
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As the ``_id`` field is present in all documents stored in MongoDB, the ``find``
```

Comment:
issue: Beginning with "As"  convolutes this sentence. I think the essence can be presented much more succinctly. Additionally, can't the `_id` field be projected away?

```suggestion
The ``find``
```